src/internal/operations/test/exchange_test.go

```diff
@@ -1157,6 +1218,15 @@ func (suite *ExchangeRestoreIntgSuite) TestRestore_Run_exchangeWithAdvancedOptio
 			GetItemsInContainerByCollisionKeyer[string](acCont),
 			collKeys[path.ContactsCategory])
 
+		currentContactIDs, err := acCont.GetItemIDsInContainer(ctx, userID, cIDs[path.ContactsCategory])
+		require.NoError(t, err, clues.ToCore(err))
+
+		assert.Equal(t, 2*len(contactIDs), len(currentContactIDs), "count of ids should be double from before")
+		for orig := range contactIDs {
+			_, ok := currentContactIDs[orig]
+			assert.True(t, ok, "original item should exist after copy")
+		}
```

Comment:
I think a call to `assert.Subset(t, currentContactIDs, contactIDs)` would work here?

Reply:
Can't do it with maps, but `maps.Keys()` works just fine.